tests/test_ssn_en_US.nim

```diff
@@ -0,0 +1,13 @@
+import std/unittest
+import faker
+import strscans
+
+test "SSN":
+  block:
+      checkpoint "it generates a valid ssn"
+      let fake = newFaker()
+      let ssn = fake.ssn()
+      let (success, aNum, gNum, sNum) = scanTuple(ssn, "$i-$i-$i")
+      assert len($aNum) == 3
```

Comment:
Please use `check` macro of `std/unittest`.

https://nim-lang.org/docs/unittest.html#check.m%2Cuntyped

```nim
check len($aNum) == 3
```